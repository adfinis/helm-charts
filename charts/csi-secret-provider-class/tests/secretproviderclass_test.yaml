suite: secretproviderclass tests
templates:
  - secretproviderclass.yaml
tests:
  - it: should match snapshot
    values:
      - ../ci/default-values.yaml
    asserts:
      - hasDocuments:
          count: 1
      - matchSnapshot: {}
