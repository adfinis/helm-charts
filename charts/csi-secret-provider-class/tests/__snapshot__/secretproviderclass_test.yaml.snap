should match snapshot:
  1: |
    apiVersion: secrets-store.csi.x-k8s.io/v1alpha1
    kind: SecretProviderClass
    metadata:
      labels:
        app.kubernetes.io/app: csi-secret-provider-class
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/version: 0.3.0
        helm.sh/chart: csi-secret-provider-class-0.3.3
      name: release-name-csi-secret-provider-class
    spec:
      parameters:
        foo: bar
      provider: fake
      secretObjects:
      - secretName: test
